ui/cap-react/src/components/drafts/form/themes/grommet/widgets/TextWidget.js

```
@@ -284,7 +282,17 @@ class TextWidget extends Component {
             this.props.options &&
             this.props.options.warningMessage
           }
+          mask={this.props.options.masked_array}
```

default mask should be empty